### PR TITLE
updates to how to test forms from charlies last feedback

### PR DESCRIPTION
**Pull request by tritland**
Removed "checkbox" text from checkbox example to avoid duplicate announcement
Updated selects to be Nato Phonetic Alphabet example instead of numbers
Added aria-invalid to last error handling example

### CODE REVIEW

File: how-to/form.md [hunk 1]
```diff
@@ -297,7 +297,7 @@ Select fields allow users to select one or more options from an expandable list
 
 <div class="how-to-test-checklist-item">
   <h3>✓ Ensure there is proper error handling for required fields</h3>
-  <p><strong>Note:</strong> When error text is not programmatically associated by use of <code>aria-describedby</code> they may not be discovered by screen reader users.</p>
+  <p><strong>Note:</strong> Error messages should be announced for screen reader users when the form recives focus. Forms should also notify screen reader users if invalid content has been entered in a field.</p>
```
Comment (joe-watkins) [inline how-to/form.md:300]: @tritland Should this be "form field receives focus" instead of just "form receives focus". Note: Typo on "recives".

Comment (tritland) [reply how-to/form.md]: thanks, updated!

**Review by jzoo — APPROVED** [review]
looks good

**Review by joe-watkins — COMMENTED** [review]
Looks good!

**Pull request merged**